### ISSUE TITLE
error during install 

### ISSUE THREAD

**Issue by sensnowy** [legotheboss/YouTube-files#47]
Hi, 
installed it and this error command keeps up popping up 

collect2: error: ld returned 1 exit status make: *** [Makefile:114: ffmpeg_g] Error 1

is eveyrthing working as planned or should i redo every step?

**Comment by micsharpe** [comment]
Same error message on the final install step.

**Comment by robynjayqueerie** [comment]
Yep, error with latest Pi OS Lite and install. Plus had many install errors doing other methods. Also loaded Ubuntu onto Pi and still can't crack it. I suspect latest ffmpeg git is not able to be compiled on Pi? Has anyone done it??
Also reduced the configure options to just sudo ./configure --arch=armel --target-os=linux and still fails.

**Comment by DRRoJohn** [comment]
Same issue. Have you find a solution of it?

**Comment by sensnowy** [comment]
Hey. No idea. It’s been so long. What I recommend is just reinstall it all. Save your lists and then reinstall. Doesn’t take long anyway and it’s faster than finding a solution.
